app/models/concerns/job_posting_searchable.rb

```diff
@@ -20,9 +19,11 @@ module JobPostingSearchable
         Rails.application.routes.url_helpers.job_posting_url(self, host: ENV['APP_HOST'], port: ENV['APP_PORT'])
       end
 
-      attribute :relevancy do
-        relevancy_for_database
-      end
+      attribute(:relevancy) { relevancy_for_database }
+
+      attribute(:description) { description.body }
+      attribute(:required_skills) { required_skills.body }
+      attribute(:preferred_skills) { preferred_skills.body }
```

Comment:
does body return with HTML tags?

Reply:
yep, checked it in local algolia index

Reply:
or do u not want the html?

Reply:
No. I don't expect to search for <ul> and actually get results

Reply:
oh well, me fixed